R/estimate_dispersion.R

```diff
@@ -47,7 +54,10 @@
 estimate_dispersion <- function(
     pt_nowcast_mat_list,
     trunc_rep_tri_list,
-    n = length(pt_nowcast_mat_list)) {
+    n = length(pt_nowcast_mat_list),
+    fun_to_aggregate = c("sum"),
+    k = 1) {
+  fun_to_aggregate <- match.arg(fun_to_aggregate)
```

Comment:
just pass in a function and have a few custom options that operate on the expected date format. Not clear why this switch approach.

Reply:
I removed it, I did set the allowable functions though? I guess we could allow min, or max but right now we can't allow mean or median because the observation model is negative binomial and so it must be integer observations.